index.html

```diff
@@ -248,70 +248,10 @@ <h4 class="date">
                                         </thead> 
                                         <tbody> 
                                             <tr>
-                                                <td class="pyred">08.30 AM</td>
-                                                <td>Registration & Breakfast</td>
-                                                <td></td>
-                                                <td></td>
-                                            </tr>
-                                            <tr>
-                                                <td class="pyred">09.00 AM</td>
-                                                <td>Introductions</td>
-                                                <td></td>
-                                                <td></td>
-                                            </tr>
-                                            <tr>
-                                                <td class="pyred">09.30 AM</td>
-                                                <td>Learn Python in Minutes</td>
-                                                <td class="pytablefont">Kracekumar Ramaraju</td>
-                                                <td class="pytablefont">Auditorium 01</td>
```

Comment:
Let's keep the `pytablefont` class for future reference with few columns for future reference. Rest looks good.